tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -424,6 +424,17 @@ public:
 // 3. The column is an RVec, the collection is not a vector
 // 4. The column is an RVec, the collection is a vector
 
+template <typename T, typename COLL>
+void FillColl(const T& v, COLL& c) {
```

Comment:
@bluehood Does the implicit cast of `T&` to `const T&` always work?

Reply:
it certainly always compile, i'm not sure whether it's always the best match when you have another overload